### PR TITLE
Add troubleshooting guide

### PR DESCRIPTION
**Pull request by prayerslayer**
People tend to run into the same set of problems.

### CODE REVIEW

File: docs/troubleshooting.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+## I can't enable or disable checks for my repository
+
+![A Github 404 API error is visible](img/troubleshooting/no-enable-disable.png)
+
+You don't have admin access for this repository. Either become an admin yourself or ask one to toggle this check.
+ 
+## I updated my Zapprfile in a PR, but it's not picked up.
+
+Zappr reads your Zapprfile from the repository's default branch (if you're not sure it's `master`). Changes on a feature branch are ignored.
```
Comment (sebastianpoeplau) [inline docs/troubleshooting.md]: On reading the sentence for the first time: "If you're not sure it's `master`" - then what? (I had to re-read to understand that "it's `master`" is the consequence.) How about rephrasing like "it's `master` by default"?

File: docs/troubleshooting.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+### The 99% case
+
+The person who enabled the Zappr check does not have access to the repository anymore, for whatever reason. Since Zappr stores a personal access token for every check (to communicate with GitHub during check execution), it's a problem if the permissions of this token change.
+ 
+**Solution**: Disable and enable the Zappr check to replace the stored token.
+
+### The case that happened only once
+
+There a limit of 1000 status updates per combination of context (like `zappr`, `zappr/pr/specification`...) and commit ID. If there are A LOT of comments in a PR, this limit might be exceeded.
```
Comment (sebastianpoeplau) [inline docs/troubleshooting.md]: Typo: "There _is_ a limit..."

File: docs/troubleshooting.md (hunk 1 above)
Comment (prayerslayer) [inline docs/troubleshooting.md]: ok

File: docs/troubleshooting.md (hunk 2 above)
Comment (prayerslayer) [inline docs/troubleshooting.md]: thanks

**Comment by sebastianpoeplau** [comment]
👍

**Pull request merged**